examples/exec_term.rs

```diff
@@ -0,0 +1,90 @@
+//! This example will spin up Zookeeper and two Kafka brokers asynchronously.
```

Comment:
nice example... can you change the description?

Reply:
Done